### PR TITLE
Card wizard

### PR DESCRIPTION
**Pull request by KAMA-Kasckak-Martin**

### CODE REVIEW

File: frontend/src/hooks/useWizards.ts [hunk 1]
```diff
@@ -19,7 +19,11 @@ export default function useWizards(){
     }
 
     function addWizard(newWizard: NewWizard) {
+
+        return axios.post("wizard-students", newWizard)
```
Comment (KAMA-Kasckak-Martin) [inline frontend/src/hooks/useWizards.ts:23]: delete this line

**Review by KAMA-Kasckak-Martin — COMMENTED** [review]
fixed code duplication

File: frontend/src/components/WizardApp.tsx [hunk 2]
```diff
@@ -2,16 +2,38 @@ import useWizards from "../hooks/useWizards";
...
+import WizardCard from "./WizardCard";
+import {Attribute1, Attribute2, Attribute3, Gender, Wizard} from "../model/Wizard";
+
+
 export default function WizardApp() {
 
     const { wizards, addWizard, removeWizard} = useWizards()
 
+
+    const wizard:Wizard = {
```
Comment (s-gbz) [inline frontend/src/components/WizardApp.tsx:14]: What is the purpose of this wizzard object?

File: frontend/src/components/WizardCard.scss [hunk 3]
```diff
@@ -0,0 +1,108 @@
+@import url('https://fonts.cdnfonts.com/css/harry-potter');
```
Comment (s-gbz) [inline frontend/src/components/WizardCard.scss:1]: In the future it would be great to have this locally

**Review by s-gbz — CHANGES_REQUESTED** [review]
Please finish the character saving functionality

**Review by licetrosales — APPROVED** [review]

**Review by KAMA-Kasckak-Martin — COMMENTED** [review]
bla bla

**Pull request merged**